lib/far_mar/market.rb

```diff
@@ -0,0 +1,72 @@
+class FarMar::Market < FarMar::Base
+
+attr_accessor :id,
+              :name,
+              :address,
+              :city,
+              :county,
+              :state,
+              :zip
+
+def initialize(attrs)
+  @id       = attrs[:id]
+  @name     = attrs[:name]
+  @address  = attrs[:address]
+  @city     = attrs[:city]
+  @county   = attrs[:county]
+  @state    = attrs[:state]
+  @zip      = attrs[:zip]
+end
+
+  def self.path
+    "/Users/Chris/Code/class_projects/farmar/far_mar/support/markets.csv"
```

Comment:
Even though this is on your computer, it's better to make these relative paths. When I ran rspec they all failed because of this. All I did to fix it was remove `/Users/Chris/Code/class_projects/farmar/far_mar/`.